### ISSUE TITLE
Adaptação do Racket para uso de código da subseção 3.3.

### ISSUE THREAD

**Issue by Caioflp** [emap-ic-2020-1/main#107]
Na subseção 3.3 do livro são introduzidos os chamados `mutators`, `set-car!` e `set-cdr!`, que são capazes de modificar a estrutura de dados `pair`. Entretanto, em Racket, pares são, por desing, imutáveis, de modo que esses `mutators` não existem. Embora haja a estrutura de dados `mutable pair` em  Racket, pequenos experimentos me mostraram que seria irritante utilizá-la (ela também não suporta `set-car!` e `set-cdr!`, além de não possuir uma forma impressa bonita).

Pesquisando um pouco, encontrei um pacote para Racket que introduz uma nova linguagem (`#lang sicp`) que contém todas as funcionalidades requeridas pelo SICP (todas mesmo, até substitui `empty` por `nil`). [Aqui](https://docs.racket-lang.org/sicp-manual/Installation.html) há um tutorial para instalá-lo. 

Para utilizar o pacote, basta iniciar o documento com `#lang sicp` ao invés de `#lang racket`.

O que você acha professor? Encontrou alguma solução melhor?

**Comment by arademaker** [comment]
não, de fato, minha preocupação com esta parte do livro é exatamente as diferenças de Racket e o Scheme usado pelo livro...eu conhecia este pacote mas não tinha testado. obrigado por apontar, acho que é nossa melhor alternativa para conseguir seguir o livro.
